include/eve/module/real/core/function/regular/generic/load.hpp

```diff
@@ -120,8 +121,48 @@ namespace eve::detail
   {
     return static_cast<T>(*p);
   }
+
+#if defined(SPY_COMPILER_IS_CLANG) or defined(SPY_COMPILER_IS_GCC)
+#define DISABLE_SANITIZERS __attribute__((no_sanitize_address)) __attribute__((no_sanitize_thread))
+#elif defined(SPY_COMPILER_IS_MSVC)
+#define DISABLE_SANITIZERS __declspec(no_sanitize_address)
+#else
+#define DISABLE_SANITIZERS
+#endif
+
+  template<typename Ptr>
+  DISABLE_SANITIZERS auto load_(EVE_SUPPORTS(cpu_), unsafe_type, Ptr ptr) noexcept
+    requires requires(Ptr ptr) { eve::load(ptr); }
+  {
+    return eve::load(ptr);
+  }
+
+  template<typename Ptr, typename Cardinal>
+  DISABLE_SANITIZERS auto load_(EVE_SUPPORTS(cpu_), unsafe_type, Ptr ptr, Cardinal const & cardinal) noexcept
+    requires requires(Ptr ptr, Cardinal cardinal) { eve::load(ptr, cardinal); }
+  {
+    using e_t = std::remove_cvref_t<decltype(*ptr)>;
+    using r_t = as_wide_t< e_t, typename Cardinal::type >;
+
+    if constexpr ( !has_aggregated_abi_v<r_t> ) return eve::load(ptr, cardinal);
+    else
+    {
+      constexpr eve::fixed< Cardinal() / 2 > half;
```

Comment:
_TODO_: this won't work for over aligned ptrs.
Also at the moment, seems like for aggregated we require the same alignment as for the smallest native. Talk to @jfalcou about it in https://github.com/jfalcou/eve/issues/495 and decide what's the right answer is.